vendor.conf

```diff
@@ -7,13 +7,13 @@ github.com/coreos/etcd 824277cb3a577a0e8c829ca9ec557b973fe06d20
 github.com/cpuguy83/go-md2man a65d4d2de4d5f7c74868dfa9b202a3c8be315aaa
 github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76
 github.com/docker/distribution b38e5838b7b2f2ad48e06ec4b500011976080621
-github.com/docker/docker 45c6f4262a865a03adaac291a9ce33c0f2190d77 
-github.com/docker/docker-credential-helpers v0.5.0
-github.com/docker/go d30aec9fd63c35133f8f79c3412ad91a3b08be06
+github.com/docker/docker c8141a1fb1ff33b2bfab85a40e5da9a282f36cdc
+github.com/docker/docker-credential-helpers v0.5.1
+github.com/docker/go d30aec9fd63c35133f8f79c3412ad91a3b08be06 #?
 github.com/docker/go-connections e15c02316c12de00874640cd76311849de2aeed5
 github.com/docker/go-events 18b43f1bc85d9cdd42c05a6cd2d444c7a200a894
 github.com/docker/go-units 9e638d38cf6977a37a8ea0078f3ee75a7cdb2dd1
-github.com/docker/libnetwork b13e0604016a4944025aaff521d9c125850b0d04
+github.com/docker/libnetwork 83e1e49475b88a9f1f8ba89a690a7d5de42e24b9
```

Comment:
Seems like libnetwork is only used for a single `IsLocalhost` function in `github.com/docker/libnetwork/resolvconf/dns`.

Could we make the vendoring more specific, and reference that package directly? Or possibly copy that function into `cli` to avoid the dependency.